docs/reference/kombu.transport.SQS.rst

```diff
@@ -22,3 +22,39 @@
     .. autoclass:: Channel
         :members:
         :undoc-members:
+
+Back-off policy
+------------------------
+Back-off policy is using SQS visibility timeout mechanism altering time diff between task retries.
+
+Configuring the queues and backoff policy::
```

Comment:
can we improve the docs more?

Reply:
Added both here and https://github.com/celery/celery/pull/6648